modules/node-observability-scripting.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="node-observability-scripting_{context}"]
+= Configuring Node Observability Operator scripting
+
+[About Scripting]
```

Comment:
I'm honestly not sure what this text is or is supposed to do, but just an FYI that if you are hoping it shows up in the rendered doc, I can't see it in the preview.

Reply:
No I added that as a draft, just for my knowledge, that will be deleted.